system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -492,15 +492,19 @@
           ;; Check that each concept was updated
           (doseq [concept-id concept-ids
                   :let [concept (-> (search/find-concepts-umm-json :collection
-                                                                   {:concept-id concept-id})
+                                                                   {:concept-id (first concept-ids)})
```

Comment:
Why the change?  We want to test each concept-id in concept-ids, now every iteration it's testing the same first concept-id?